docs/oid-info.md

```diff
@@ -23,35 +23,49 @@ The `.1` is added to the root OID for sigstore for all OIDs set by Fulcio.
 This contains the `issuer` claim from the OIDC Identity Token that was
 presented at the time the code signing certificate was requested to be created.
 This claim is the URI of the OIDC Identity Provider that digitally signed the
-identity token.
+identity token. For example: https://token.actions.githubusercontent.com.
 
-### 1.3.6.1.4.1.57264.1.2 | GitHub Workflow Trigger
+### 1.3.6.1.4.1.57264.3.1 | Source URI
 
-This contains the `event_name` claim from the GitHub OIDC Identity token that
-contains the name of the event that triggered the workflow run.
-[(docs)][github-oidc-doc]
+Should include a fully qualified source repository URL that the build was based
+on.
 
-### 1.3.6.1.4.1.57264.1.3 | GitHub Workflow SHA
+### 1.3.6.1.4.1.57264.3.2 | Source Digest
 
-This contains the `sha` claim from the GitHub OIDC Identity token that contains
-the commit SHA that the workflow run was based upon. [(docs)][github-oidc-doc]
+An immutable reference to a specific version of the source code that the build
+was based upon. Should include the digest type followed by the digest, e.g.
+`sha1:abc123`.
 
-### 1.3.6.1.4.1.57264.1.4 | GitHub Workflow Name
+### 1.3.6.1.4.1.57264.3.3 | Source Ref
 
-This contains the `workflow` claim from the GitHub OIDC Identity token that
-contains the name of the executed workflow. [(docs)][github-oidc-doc]
+The source ref that the build run was based upon. For example: `refs/head/main`.
 
-### 1.3.6.1.4.1.57264.1.5 | GitHub Workflow Repository
+### 1.3.6.1.4.1.57264.3.4 | Build Config URI
```

Comment:
I have a question about what to do with external modules. Let's take GHA as an example, one can use an external action and reference by a branch and not an presumably immutable SHA or tag:

```yaml
uses: publisher-username/repo-name@branch-name
```

In that case the `branch-name` will be evaluated at the time of build execution. Similar example is possible with CircleCI Orbs and with other CI systems.

I wonder if there is a need to have URIs to all the config files that affected the build. Or should this transitive nature be reflected in the following "Build Config Digest"?

Reply:
Interesting point, I don't think we have any good way of capturing what exact commit external actions where pulled from in GitHub Actions but this would be nice to know for sure. 

My thinking here was that `Build Config` would only refer to the initiating or top-level config (or entry point in SLSA lingo) that was used when running the job in response to the "Build Trigger". The digest would be the commit sha of this top-level config. Given this you could see the exact top-level workflow that was used but not what exact transitive actions where used unless you did some kind of date based commit lookup in each action at the time the job ran?